### PR TITLE
Added dependencies for App Insights telemetry capture

### PR DESCRIPTION
**Pull request by coderandhiker**
Added dependencies for App Insights telemetry capture

### CODE REVIEW

**Review by twsouthwick — COMMENTED** [review]
Can you hook up appinsights to the ilogger stuff?

**Comment by coderandhiker** [comment]
> Can you hook up appinsights to the ilogger stuff?

Added App Insights to the ILoggingBuilder

File: src/WWTMVC5/Global.asax.cs [hunk 1]
```diff
@@ -133,6 +133,7 @@ private static IServiceProvider BuildServiceProvider()
             {
                 builder.AddFilter("Swick.Cache", LogLevel.Trace);
                 builder.AddDebug();
+                builder.AddApplicationInsights();
```
Comment (twsouthwick) [inline src/WWTMVC5/Global.asax.cs:136]: I love how simple this is!

**Review by twsouthwick — APPROVED** [review]

**Review by pkgw — APPROVED** [review]
Straightforward indeed!

**Pull request merged**